v2/cmd/integration-test/dns.go

```diff
@@ -59,6 +60,22 @@ func (h *dnsCAA) Execute(filePath string) error {
 	return expectResultsCount(results, 1)
 }
 
+type dnsTLSA struct{}
+
+// Execute executes a test case and returns an error if occurred
+func (h *dnsTLSA) Execute(filePath string) error {
+	var routerErr error
+
+	results, err := testutils.RunNucleiTemplateAndGetResults(filePath, "google.com", debug)
```

Comment:
Could you recommend any valid domain with TLSA record?

Reply:
I could not find it right now